README.md

```diff
@@ -349,6 +361,32 @@ created.
 
 Example: `queue.create_tube('list_of_sites', 'fifo', {temporary = true})`
 
+## Set queue settings
+
+```lua
+queue.set_settings({options})
+```
+
+Set queue settings.  
+In case of failure, an error will be thrown.
+Available `options`:
+`ttr` - time to release in seconds. The time after which, if there is no active
```

Comment:
I think it should be a list. Currently it looks quite strange:
![image](https://user-images.githubusercontent.com/8830475/102108207-afe9d400-3e43-11eb-9b36-aefed3e2a0f7.png)

Reply:
Done.